### DISCUSSION TITLE
expression distribution of genes in different clusters

### DISCUSSION THREAD

**Discussion by yihancc** [brianhie/scanorama#119] [Q&A] [answered]
Hi @brianhie, thanks for this wonderful tool!

I use integrate_scanpy in scanorama to process my datasets,  and notice that some genes are expressed in several clusters, but gathered in one cluster  when processed by other tools such as BBKNN and scvi. 
![image](https://user-images.githubusercontent.com/102579767/160569780-03af33e3-ef3f-4817-aa13-2b0c59146318.png)

I tried different knn value but got similar reults. 

My single-cell data consists of 5 public datasets, 37 samples. All ~70,000 cells are of the same type in the same tissue but from different age groups from fetuses to adults. 

I wonder whether this happens due to the biological variance? How can I understand and deal with it?

**Comment by brianhie** [comment, marked as answer]
That looks quite interesting -- especially how different the overall results are between methods. Without any additional information, I'm not quite sure what is going on. One good thing to check with integration methods is to look at two known biological replicates separated by batch, and to see if that variance is removed while preserving the within-replicate biological variance.

**Reply by yihancc** [reply]
Thanks for your reply! 

I tried several trials with a single sample from different age groups and found that at some age points there are actually two or more clusters that are far away from each other that also express gene A! 

While I do not know why the clusters are so far apart, I believe the above results processed by Scanorama are largely based on biological variance.

**Reply by brianhie** [reply]
Good to hear, yes it's definitely good practice to include controls when performing integration
